About.html

```diff
@@ -0,0 +1,89 @@
+            <a href="#">Whatsap</a>
+            <a href="#">training</a>
+            <a href="#">My page</a>
+            <a href="#">logout</a>
+        </nav>
+        <nav class="navlogo">
+            <a class="img-logo" href="index.html"><img src="img-folder/logo-image---00.png" alt=""></a>
+            <div class="hamburg">
+                <div class="hamburg1"></div>
+                <div class="hamburg2"></div>
+                <div class="hamburg3"></div>
+            </div>
+            <ul class="menu">
+                <li><a class="menu-item" href="index.html">home</a></li>
+                <li><a class="menu-item" href="About.html">About</a></li>
+                <li><a class="menu-item" href="#logo">logo</a></li>
+                <li><a class="menu-item" href="#vision">Vision</a></li>
+            </ul>    
+        </nav>
+    </header>    
+        <section class="grtittle">
+            <h1>Kamulete training center(Rwanda)</h1>
+            <hr class="linebar">
+            <div class="centerpar">
+                <p>kamulete trainig center for all people around the world,<br> we invite you to join our training center.
+                    kamulete training center was created in 2023 by kamulete bertin who is the CEO form Rwanda,<br> he found that to train people by knwowing many languages is very helps full and develops creativness in someone.
+                    with that this trainig center give you a diploma after complited the month acording to your program choosen,<br> that diploma is valid in each country around the word.
+                    we have more than 20 teachers by which we have collaboration and they helps us to give some online classes.
+                </p>
+            </div>
+          <div class="contactclass">
+            <p class="contactclass1">please contact us via email for more information.    </p>
+            <p class="contactclass2">email: Kamuletetrainingcenter988@gmail.com<br></p>
+            <p class="contactclass3">@250 kigali , palace (Rwanda)</p>
+          </div>
+        </section>
+        <section id="logo" class="classlog">
+            <h2>kamulete-training-center-logo</h2>
+            <hr class="linebar">
+            <p>the logo of kamulete traning center logo was decided from a competition from 12 january to 30january.
+                After finishing the competition voting and selection committe's judjjing process,<br> the logo from 20 years oldswon the prize-winning-logo for kamulete training center as follow
+            </p>
+            <img src="img-folder/logo-image.png" alt="">
+        </section>
+        <section id="vision" class="vissonsee">
+            <h2>see our vision and goals </h2>
+            <hr class="linebar">
+            <p>see how we working with our students</p>
+           <div>
+           <p class="section-red">Different community</p>
+            <p class="section-red1">Different community</p>
+            </div>
```

Comment:
- [ ] Please make sure to add `margin` from the left and right of the screen of the `section-red` and `section-red1` to make it look like as in the design guidelines

Your design:

<img width="326" alt="Screenshot 2023-03-16 at 9 55 58 PM" src="https://user-images.githubusercontent.com/44798044/225729289-cfe68249-bdd9-44cc-8505-170ae5b4903a.png">

Design guideline:

<img width="338" alt="Screenshot 2023-03-16 at 9 49 20 PM" src="https://user-images.githubusercontent.com/44798044/225729304-9d3ac83c-94a6-4172-a40a-22b3e0a6a224.png">